_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Software/Developer Tools 
+layout: plain
+redirect_from:
```

Comment:
This you do not need. It exists for a few of the current WG pages as at one point in the past they were hosted under `activities` and then we changed it to `working groups`, but for any new pages it's unnecessary.

Reply:
Done.